xmppserver/src/main/webapp/muc-room-clear-chat.jsp

```diff
@@ -60,9 +60,13 @@
     if (clear) {
         // Clear the room
         if (room != null) {
-            room.clearChatHistory();
-            // Log the event
-            webManager.logEvent("clear chat history of MUC room ", roomName);
+            webManager.logEvent("Making a request to clear chat history of MUC room ", roomJID.toString());
```

Comment:
These events are logged in an audit log, not a 'regular' (debug) log. Please do not add two messages, just one will be fine.

Reply:
Done.